shared/openId4VCI/Utils.ts

```diff
@@ -0,0 +1,86 @@
+import {ENABLE_OPENID_FOR_VC} from 'react-native-dotenv';
```

Comment:
Do we need to have separate util for openId4VCI or it can be part of existing util?